.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,103 @@
+---
+name: E2E Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths-ignore:
+      - .circlecli/**
+      - meta/**
+      - .gitignore
+      - .markdownlint.yml
+      - .yamllint
+      - ansible.cfg.galaxy
+      - CHANGELOG.md
+      - LICENSE
+      - poetry.lock
+      - pyproject.toml
+      - README.md
+      - Vagrantfile
+  push:
+    branches:
+      - master
+      - ansible-molecule-4
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+env:
+  PY_COLORS: true
+  ANSIBLE_FORCE_COLOR: true
+  MOLECULE_DEBUG: false
+  MOLECULE_VERBOSITY: 0
+  MOLECULE_NO_LOG: true
+  MOLECULE_VERSION: "3.6.1"
+  MOLECULE_DOCKER_VERSION: "1.1.0"
+
+concurrency:
+  group: e2e
+  cancel-in-progress: false
+
+jobs:
+  molecule:
+    runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        st2_repo:
+          - stable
+          - unstable
+        os:
+          - centos7
+          - ubuntu18
+          - rockylinux8
+          - ubuntu20
+        ansible-core:
+          - "2.11.12"
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python.version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python.version }}
```

Comment:
Python is not in the matrix anymore, worth removing the `${{ matrix.python.version }}`